### PR TITLE
Add Proper License to com.jme3.anim files

### PR DESCRIPTION
**Pull request by ItsMike54**
In order to chip away at this issue, https://github.com/jMonkeyEngine/jmonkeyengine/issues/1001
I've added the proper jME license to the Anim files written by @Nehon (or at least the ones that he committed to, excluding the ones by pspeed which already have the new licenses) . 
Now in the previous discussion on that issue, it seemed as if @stephengold was attempting to contact nehon to get his permission. If Nehon replies, then we'll follow from there, but if he doesn't, I say that since he contributed as an engine leader, it would be fine to go ahead and add the licenses.

### CODE REVIEW

File: jme3-core/src/main/java/com/jme3/anim/SkinningControl.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009-2019 jMonkeyEngine
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/SkinningControl.java:2]: This change seems unnecessary.

File: jme3-core/src/main/java/com/jme3/anim/AnimTrack.java [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/AnimTrack.java:31]: Nehon didn't claim authorship of this file.

File: jme3-core/src/main/java/com/jme3/anim/ArmatureMask.java [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/ArmatureMask.java:31]: Nehon didn't claim authorship of this file.

File: jme3-core/src/main/java/com/jme3/anim/AnimTrack.java (hunk 2 above)
Comment (ItsMike54) [reply jme3-core/src/main/java/com/jme3/anim/AnimTrack.java:31]: Ohh....well should that be removed then? Or try to find the author?

File: jme3-core/src/main/java/com/jme3/anim/MatrixJointModelTransform.java [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/MatrixJointModelTransform.java:31]: Nehon didn't claim authorship of this file.

File: jme3-core/src/main/java/com/jme3/anim/SeparateJointModelTransform.java [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/SeparateJointModelTransform.java:31]: Nehon didn't claim authorship of this file.

**Comment by ItsMike54** [comment]
@stephengold , simply remove those headers then? Or try to find their author?

File: jme3-core/src/main/java/com/jme3/anim/Weights.java [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/Weights.java:31]: Nehon didn't claim authorship of this file.

File: jme3-core/src/main/java/com/jme3/anim/tween/ContainsTweens.java [hunk 7]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/tween/ContainsTweens.java:31]: Nehon didn't claim authorship of this file.

File: jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java [hunk 8]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java:31]: Nehon didn't claim authorship of this file.

File: jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java [hunk 9]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java:31]: Nehon didn't claim authorship of this file.

**Comment by stephengold** [comment]
> @stephengold , simply remove those headers then? Or try to find their author?

I don't know what to do with these files. Their style matches Nehon's, so I'm 90% sure he wrote them. But he's the only person who knows for sure, and as I said, he hasn't responded to my inquiries. Perhaps someone else will have better luck contacting him.

**Comment by ItsMike54** [comment]
Well assumptions bite me in the ass, and I'll remove those right now.

File: jme3-core/src/main/java/com/jme3/anim/AnimTrack.java [hunk 10]
```diff
@@ -8,5 +8,4 @@
     public void getDataAtTime(double time, T store);
     public double getLength();
 
-
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/AnimTrack.java:11]: I don't see any reason to modify AnimTrack.java

File: jme3-core/src/main/java/com/jme3/anim/tween/action/BlendAction.java [hunk 11]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/BlendAction.java:31]: Nehon didn't claim authorship.

File: jme3-core/src/main/java/com/jme3/anim/tween/action/BlendSpace.java [hunk 12]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/BlendSpace.java:31]: Authorship uncertain.

File: jme3-core/src/main/java/com/jme3/anim/tween/action/BlendableAction.java [hunk 13]
```diff
@@ -1,3 +1,34 @@
...
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (stephengold) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/BlendableAction.java:31]: Authorship uncertain.

**Comment by ItsMike54** [comment]
@stephengold , I went ahead and reviewed the files, removing all that Nehon didn't expressly state his authorship in. I did add licenses to some that I missed previously that he did state authorship of.

**Comment by stephengold** [comment]
Please take another look at LinearBlendSpace.java

**Comment by ItsMike54** [comment]
Obviously I can't seem to make a perfect PR just yet, but LinearBlendSpace.java is fixed.

**Comment by stephengold** [comment]
As far as I can tell, this PR just needs the one-line change to AnimTrack.java and then it will be ready to merge.

@ItsMike54 have you abandoned this PR?

**Comment by ItsMike54** [comment]
@stephengold Over the past week I've been trying to make progress on a project and I don't have much time to do so, so I abandoned it for that time.
But since it was brought up again, I reverted the AnimTrack.java interface back to its original state before I modified it.

**Comment by stephengold** [comment]
It's not back to its original state.

**Comment by ItsMike54** [comment]
@stephengold , I deleted the AnimTrack.java from my branch and uploaded the AnimTrack.java interface that is currently on Master jME branch. That file is completely unmodified, and I don't know what else to say about it.

**Pull request merged**